src/components/DeletionSwipeable.tsx

```diff
@@ -34,13 +34,14 @@ interface DeletionSwipableProps {
 
 const rightThreshold = 40
 
-const DeletionSwipable = ({ apiKey, refresh, children }: DeletionSwipableProps): ReactElement => {
+const DeletionSwipeable = ({ apiKey, refresh, children }: DeletionSwipableProps): ReactElement => {
   const [isModalVisible, setIsModalVisible] = useState<boolean>(false)
-  const swipeableRow: RefObject<Swipeable> = React.createRef<Swipeable>()
+  const swipeableRow: RefObject<Swipeable> = useRef<Swipeable>(null)
 
   useEffect(() => {
-    swipeableRow.current?.close()
-  }, [swipeableRow])
+    // closes the swipeable when leaving the screen
+    return swipeableRow.current?.close()
+  })
```

Comment:
No dependencies here?

Reply:
No when using `useRef` the object is mutable and therefore never changes and no dependency.